modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -7252,6 +7252,8 @@ feature.flag.COMMERCE-12754.description=Users can issue refunds to an order's or
 feature.flag.COMMERCE-12754.title=Refund Order Payments
 feature.flag.COMMERCE-13024.description=Users can view and edit custom object fields for users and accounts via the Organization Management widget.
 feature.flag.COMMERCE-13024.title=Organization Management Widget and Custom Object Fields
+feature.flag.LPD-8012.description=Send customized Publication information to users.
+feature.flag.LPD-8012.title=Register Publication notifications
```

Comment:
Leave out the FF title and description for now. We'll add it when Laszlo have these defined.